src/js/modules/wallet/controllers/settings/security/settings.security.ctrl.js

```diff
@@ -0,0 +1,513 @@
+angular.module("blocktrail.wallet")
+    .controller("SettingsSecurityCtrl", SettingsSecurityCtrl);
+
+// TODO Review this part, decrease dependencies, create form settings service and move $http request to service
+function SettingsSecurityCtrl($scope, $http, $rootScope, $q, cryptoJS, sdkService, launchService, activeWallet,
+                            $translate, $timeout, $log, $sce, dialogService, accountSecurityService, $filter,
+                            CONFIG, $modal, formSettingsService, passwordStrengthService, settingsService) {
+
+    var settings = settingsService.getReadOnlySettingsData();
+
+    var listenerEnabled2faToggle;
+    // Final 2fa true/false
+    var isEnabled2fa = false;
+    // Toggle for enable in scope
+    $scope.isEnabled2fa = false;
+
+    // TODO: Show information about email verification
+    $scope.hasEmailChanged = false;
```

Comment:
unused

Reply:
whoops yes, only belongs to profile ctrl